src/ActionModal/components/UndelegationModal.vue

```diff
@@ -18,6 +18,18 @@
     @close="clear"
     @txIncluded="onSuccess"
   >
+    <TmFormGroup
+      v-if="session.addressRole === `stash`"
+      class="action-modal-form-group"
+    >
+      <div class="form-message notice">
+        <span>
+          This is a stash account, you can decrease the amount you stake but you
```

Comment:
@mariopino this is wrong, right?

Reply:
It is outdated. Now it reads "This is a stash account, you cannot perform any unstake related
          action. To decrease the amount your stake and change your validators
          you need to sign in with your controller account"

Reply:
Maybe you can change nominations though 🤔 Now thinking about it

Reply:
I am going to test